ruby-builder-versions.js

```diff
@@ -10,7 +10,7 @@ export function getVersions(platform) {
       "head"
     ],
     "jruby": [
-      "9.2.9.0", "9.2.10.0", "9.2.11.0",
+      "9.1.17.0", "9.2.9.0", "9.2.10.0", "9.2.11.0",
```

Comment:
Could you add `9.1.17.0` on its own line, much like 2.4.x vs 2.5.x in MRI above?